scripts/authors.scala

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+exec scala "$0" "$@"
+!#
+
+/*
+ * Usage:
+ *    authors.scala tag1 tag2
+ *
+ * or if on non unixy os:
+ *    scala authors.scala tag1 tag2
+ *
+ * requires scala 2.11.x+ and command line git on path
```

Comment:
Do you want to make that change in this PR @johanandren ? :)
Sounds like a good idea 👍